.env.example

```
@@ -16,6 +18,12 @@ OPENCODE_API_KEY=
 OPENCODE_MODEL=opencode-go/deepseek-v4-pro
 OPENCODE_CONTEXT_WINDOW=131072
 
+# OpenAI по подписке ChatGPT (если MODEL_PROVIDER=codex). Вход: `iva login`
+# (по ссылке+коду) или `iva login --browser`. Токен → data/codex-auth.json (0600, не в .env).
+# API-ключ НЕ нужен. Модель выбирается в `iva config` из реального списка подписки.
+CODEX_MODEL=gpt-5.1
+CODEX_CONTEXT_WINDOW=272000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CODEX_CONTEXT_WINDOW key should go before the CODEX_MODEL key

(UnorderedKey)